cpp/src/io/parquet/writer_impl.cu

```diff
@@ -1923,12 +1923,12 @@ auto convert_table_to_parquet_data(table_input_metadata& table_meta,
         max_write_size = std::max(max_write_size, ck.compressed_size);
 
         if (ck.ck_stat_size != 0) {
-          column_chunk_meta.statistics_blob.resize(ck.ck_stat_size);
-          CUDF_CUDA_TRY(cudaMemcpyAsync(column_chunk_meta.statistics_blob.data(),
-                                        dev_bfr,
-                                        ck.ck_stat_size,
-                                        cudaMemcpyDefault,
-                                        stream.value()));
+          std::vector<uint8_t> stats_blob;
+          stats_blob.resize(ck.ck_stat_size);
+          CUDF_CUDA_TRY(cudaMemcpyAsync(
+            stats_blob.data(), dev_bfr, ck.ck_stat_size, cudaMemcpyDefault, stream.value()));
```

Comment:
This can be replaced by `make_host_vector_sync` with input is a `host_span`.

Reply:
Thanks. It should be `make_std_vector_sync(device_span`